target/cheri-common/cheri_utils.h

```diff
@@ -107,6 +107,19 @@ static inline int64_t cap_get_otype(const cap_register_t* c) {
     return result < CAP_LAST_NONRESERVED_OTYPE ? result : result - CAP_MAX_REPRESENTABLE_OTYPE - 1;
 }
 
+static inline bool cap_exactly_equal(const cap_register_t *cbp, const cap_register_t *ctp)
+{
+    return !!(cbp->cr_tag           == ctp->cr_tag           &&
```

Comment:
Why?! Everything in here is a bool already.

Reply:
Copy-paste.